test/puppetlabs/puppetdb/scf/storage_test.clj

```diff
@@ -1394,23 +1394,23 @@
       (with-db-version db version
         (fn []
           (is (= result (db-unsupported-msg)))))
-      "PostgreSQL" [8 1] "PostgreSQL DB versions 9.2 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
-      "PostgreSQL" [8 2] "PostgreSQL DB versions 9.2 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
-      "PostgreSQL" [8 3] "PostgreSQL DB versions 9.2 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
-      "PostgreSQL" [8 4] "PostgreSQL DB versions 9.2 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
-      "PostgreSQL" [9 0] "PostgreSQL DB versions 9.2 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
-      "PostgreSQL" [9 1] "PostgreSQL DB versions 9.2 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
-      "PostgreSQL" [9 2] "PostgreSQL DB versions 9.2 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
-      "PostgreSQL" [9 3] nil
+      "PostgreSQL" [8 1] "PostgreSQL DB versions 9.3 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
+      "PostgreSQL" [8 2] "PostgreSQL DB versions 9.3 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
+      "PostgreSQL" [8 3] "PostgreSQL DB versions 9.3 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
+      "PostgreSQL" [8 4] "PostgreSQL DB versions 9.3 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
+      "PostgreSQL" [9 0] "PostgreSQL DB versions 9.3 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
+      "PostgreSQL" [9 1] "PostgreSQL DB versions 9.3 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
+      "PostgreSQL" [9 2] "PostgreSQL DB versions 9.3 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
+      "PostgreSQL" [9 3] "PostgreSQL DB versions 9.3 and older are no longer supported. Please upgrade Postgres and restart PuppetDB."
```

Comment:
Wonder if "versions older than 9.4 are no longer supported" might be a touch clearer.